tmt/utils.py

```diff
@@ -1792,6 +1798,13 @@ def to_serialized(self) -> Dict[str, Any]:
 
         fields = self.to_dict()
 
+        for name, _ in fields.items():
```

Comment:
Why not use `fields.keys()` directly?

Reply:
As some keys might be not worth serializing, by inspecting the serialized collection there's no risk of adding a key that's supposed to be not there. Probably not happing now in our code, but a custom `to_dict()` might decide to "hide" some of the keys.

Reply:
I meant why not just check keys of the dictionary:

```suggestion
        for name in fields.keys():
```

Reply:
Aah, right. Sorry, fixed in 19917f32.